deprecated/PI_CloudAhoyWriter.py

```diff
@@ -46,7 +46,6 @@ def XPluginStart(self):
 		self.IsLogging = False
 		self.InitDatarefs()
 		self.MenuSetup()
-		self.StartLogging()
```

Comment:
I accidently left this change in my pull request. I didn't like it when it started logging automatically.